components/Common/Pagination/PaginationListItem/index.module.css

```diff
@@ -0,0 +1,16 @@
+.listItem {
+  @apply flex
+  h-10
+    w-10
+    items-center
+    justify-center
+    rounded
+    px-3
+    py-2.5
+    !text-neutral-800
```

Comment:
Don't forget the `.listeItem a` thingy that I've mentioned here.

Reply:
It did not work, as I had already set `.listItem` to the anchor, but that was not working... What ended up working was to target the same pseudo-classes as in the base.css styles:

```css
.listItem,
.listItem:link,
.listItem:active {
  @apply aria-current:bg-green-600
    aria-current:text-white
    flex
    h-10
    w-10
    items-center
    justify-center
    rounded
    px-3
    py-2.5
    text-neutral-800
    hover:bg-neutral-100
    dark:text-neutral-200
    hover:dark:bg-neutral-900;
}

```